### PR TITLE
docs: Clarify naming of KMS to AWS KMS

### PR DESCRIPTION
**Pull request by acioc**
_Issue #, if available:_
* https://github.com/awslabs/aws-encryption-sdk-specification/issues/67

_Description of changes:_
* Clarify naming of KMS to AWS KMS

_Follow Ups:_
* Move kms-keyring file to aws-kms-keyring

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: framework/keyring-interface.md [hunk 1]
```diff
@@ -5,10 +5,14 @@
...
 ## Version
 
-0.2.0
+0.3.0
 
 ### Changelog
 
+- 0.3.0
+
+  - Clarify naming of KMS to AWS KMS
```
Comment (mattsb42-aws) [inline framework/keyring-interface.md:14]: Let's link to the issue here.

File: framework/kms-keyring.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 ## Version
 
-0.2.0
+0.3.0
 
 ### Changelog
 
+- 0.3.0
+
+  - Clarify naming of KMS to AWS KMS
```
Comment (mattsb42-aws) [inline framework/kms-keyring.md:14]: Let's link to the issue here.

File: framework/structures.md [hunk 3]
```diff
@@ -5,10 +5,14 @@
...
 ## Version
 
-0.2.0
+0.3.0
 
 ### Changelog
 
+- 0.3.0
+
+  - Clarify naming of KMS to AWS KMS
```
Comment (mattsb42-aws) [inline framework/structures.md:14]: Let's link to the issue here.

**Comment by mattsb42-aws** [comment]
Thinking about this also, I think that these should be Z increments, not Y. I would classify this as a "fix", not a "feature".

File: framework/keyring-interface.md (hunk 1 above)
Comment (acioc) [reply framework/keyring-interface.md:14]: Sure!

File: framework/structures.md (hunk 3 above)
Comment (acioc) [reply framework/structures.md:14]: Updated

File: framework/kms-keyring.md (hunk 2 above)
Comment (acioc) [reply framework/kms-keyring.md:14]: Updated

**Comment by acioc** [comment]
> Thinking about this also, I think that these should be Z increments, not Y. I would classify this as a "fix", not a "feature".

That's fair - I wasn't really sure how we handle versioning here. Updating to 0.2.1 instead of 0.3.0

**Comment by mattsb42-aws** [comment]
> I wasn't really sure how we handle versioning here

Yup; that's on me :) #35

**Comment by acioc** [comment]
Merging this and fixing the merge conflicts with https://github.com/awslabs/aws-encryption-sdk-specification/pull/123

Will follow up once https://github.com/awslabs/aws-encryption-sdk-specification/pull/123 is merged with a PR moving the AWS KMS keyring page to aws-kms-keyring.md

**Pull request merged**